### ISSUE TITLE
Possible to get device info (battery level) from API?

### ISSUE THREAD

**Issue by jat255** [briis/weatherflow_forecast#9]
I may be missing it, but I don't see any sensors created with this integration for device info. In particular, I'm interested in tracking the battery level.

In playing with the API at https://weatherflow.github.io/Tempest/api/swagger, this info appears to be available at the `/observations/device/{device_id}` endpoint. A request like this returns the battery level (among other things)

`curl -X GET 'https://swd.weatherflow.com/swd/rest/observations/device/{device_id}?token=xxxxxx'`

```json
{
  "status": {
    "status_code": 0,
    "status_message": "SUCCESS"
  },
  "device_id": xxxxxx,
  "type": "obs_st",
  "source": "cache",
  "summary": {
    "pressure_trend": "steady",
    "strike_count_1h": 0,
    "strike_count_3h": 0,
    "precip_total_1h": 0.0,
    "precip_accum_local_yesterday": 0.0,
    "precip_accum_local_yesterday_final": 0.0,
    "precip_analysis_type_yesterday": 0,
    "feels_like": 19.9,
    "heat_index": 19.9,
    "wind_chill": 19.9
  },
  "obs": [
    [
      1696997355,
      0.07,
      0.94,
      1.8,
      234,
      3,
      821.6,
      19.9,
      20,
      0,
      0,
      0,
      0,
      0,
      0,
      0,
      2.62,   <--- this is the battery level
      1,
      0,
      0,
      0,
      0
    ]
  ]
}

```

**Comment by briis** [comment]
You are right. It is not part of this. I try to avoid making too many calls to server, and currently we do two, to get forecast and weather data. I will consider if making a third call, to get device info like battery is worth it.

**Comment by jamesfk** [comment]
+1 for this with a computed battery percentage, if possible, please. I moved over the the official local integration but miss many of the computed sensors so have this running as well now to get the best of both worlds.

**Comment by briis** [comment]
Now released in v0.3.0 which was just pushed out. Please re-open this issue if it does not work for you.
